frontend/src/JumpToOffset.svelte

```diff
@@ -0,0 +1,68 @@
+<style>
+  input {
+    margin-bottom: 1em;
+    width: 125%;
+    background: inherit;
+    color: inherit;
+    border: none;
+    text-align: center;
+  }
+
+  input:focus {
+    outline: none;
+    box-shadow: inset 0 -1px 0 var(--main-fg-color);
+  }
+</style>
+
+<script>
+  import { calculator } from "./helpers";
+  import { onMount, tick } from "svelte";
+
+  export let dataPromise, scrollY;
+  let startOffset,
+    input,
+    mounted = false;
+  const alignment = 16;
+
+  let dataLength = 0;
+  dataPromise.then((data) => {
+    dataLength = data.byteLength;
+  });
+
+  onMount(() => {
+    mounted = true;
+  });
+
+  $: if (mounted) {
+    startOffset = Math.max(
+      Math.ceil((dataLength * $scrollY.top) / alignment) * alignment,
+      0
+    );
+    input.value = `0x${startOffset.toString(16)}`;
+  }
+</script>
+
+<input
+  type="text"
+  on:focusout="{() => {
```

Comment:
I don't think jumping on focusout is a good idea. This means there's no way to "cancel" jumping if you start typing then change your mind, right? I think just waiting on an Enter press is perfectly sufficient.

Reply:
Well you could write something and <kbd>Ctrl</kbd>+<kbd>z</kbd>, but that you make a fair point about wanting to undo. I could make it only submit on <kbd>Enter</kbd>, but that means if you focus out and scroll a little, what you wrote will be overwritten.

Reply:
While I think I'd personally prefer it to jump on focusout, I've changed it for now. If people express a different preference, we can always do it that way later.